packages/adapter-cloudflare-workers/files/entry.js

```diff
@@ -39,10 +39,31 @@ async function handle(event) {
 		});
 
 		if (rendered) {
-			return new Response(rendered.body, {
-				status: rendered.status,
-				headers: rendered.headers
-			});
+			if (
+				rendered.body &&
+				typeof rendered.body === 'object' &&
+				typeof rendered.body[Symbol.asyncIterator] === 'function'
+			) {
+				const body_stream = new ReadableStream({
+					async pull(controller) {
+						const { value, done } = await rendered.body.next();
+						if (done) {
+							controller.close();
+						} else {
+							controller.enqueue(value);
+						}
+					}
+				});
```

Comment:
Wouldn't we need something like `cancel() { return rendered.body.return(); },` here?

Reply:
Yes, this section needs more hardening. Will make a pass at all edge cases.